pmd-java/src/main/java/net/sourceforge/pmd/lang/java/internal/JavaAstProcessor.java

```diff
@@ -162,13 +165,13 @@ public static JavaAstProcessor create(ClassLoader classLoader,
         );
     }
 
-    private static void bench(String label, Runnable runnable) {
+    public static void bench(String label, Runnable runnable) {
```

Comment:
Maybe these two methods (`bench(...)`) can stay private again?

Update: turns out, no: It is called via SymbolTableHelper::earlyDisambig -> InternalApiBridge::disambig -> AstDisambiguationPass::disambig

Maybe something to improve later...